tools/build-react

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Rudimentary safety check.
+[[ ! -f "tools/build-react" ]] && echo 'Please run this from package root!' && return 1;
+
+# Define some paths.
+src_react_dir=src/react
+src_css_dir=src/css
+dest_react_dir=build/react
+
+# Make dir if it doesn't exist.
+mkdir -p $dest_react_dir
+
+# Copy source JSX as well.
+cp "$src_react_dir"/*.jsx "$dest_react_dir"
```

Comment:
My reasoning behind the `.jsx` copy was two-fold:

1.  Keep those interested only in the end result out of the `src` folder.
2. Keeping the generated React data file in the build folder rather than dumping it in the `src` folder. To me it looked odd to have in `build` one lonely generated `.jsx` file with its corresponding `.js`, and the other two `.js` files without their `.jsx`.

As for other stuff that's copied into the build dir: where would you put `example.html` and `example.css` instead?

Reply:
Looks like `example.html` is built, so it would have to go in the build folder.

The argument against separating the things in both of these cases is a good one.

Reply:
> Looks like `example.html` is built, so it would have to go in the build folder.

Ah, yes. There used to be a second one, but that changed after switching to `wawoff2`. The remaining one is built, so is a non-issue.